app/services/upsert.js

```diff
@@ -0,0 +1,9 @@
+const { User } = require('../models');
```

Comment:
Create a `services/users.js` file and put all the user operations there

Reply:
Done